Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb

```diff
@@ -0,0 +1,1041 @@
+{
```

Comment:
<p>Could you also have a conclusion part as in #83 to explain the readers that one of the accuracies is the train accuracy, maybe @Avinashshah099&nbsp;could help in this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>